src/Compilers/Core/Portable/MetadataReader/MetadataDecoder.cs

```diff
@@ -1096,14 +1153,19 @@ private void DecodeParameterOrThrow(ref BlobReader signatureReader, /*out*/ ref
             bool refersToNoPiaLocalType;
 
             SignatureTypeCode typeCode;
-            info.CustomModifiers = DecodeModifiersOrThrow(ref signatureReader, out typeCode);
+            info.CustomModifiers = DecodeModifiersOrThrow(ref signatureReader, out typeCode, IsAcceptedIsConstModifierType, out bool isConstFound);
```

Comment:
I like the enum suggestion. will switch to it now.

---
In reply to: [118755259](https://github.com/dotnet/roslyn/pull/19658#discussion_r118755259) [](ancestors = 118755259)